cores/arduino/stm32/usb/cdc/usbd_cdc_if.h

```diff
@@ -50,6 +50,7 @@ void CDC_continue_transmit(void);
 void CDC_resume_receive(void);
 void CDC_init(void);
 void CDC_deInit(void);
+uint8_t CDC_connected();

```

Comment:
Maybe use a `bool `return should be better.
```suggestion
bool CDC_connected();
```

Reply:
Sorry, I'm going to revert your suggestion. AFAIK bool is a C++ type and this is C file.
_Bool is available in C99 compilers and allegedly there is a macro defining bool as _Bool, but not in older C89/C90.

Reply:
bool is used in several other c files (including usbd_cdc_if.c) so this will not be an issue.